katello/katello-change-hostname

```diff
@@ -217,6 +217,10 @@ STDOUT.puts installer_output
 STDOUT.puts "Regenerating ueber certs"
 `foreman-rake katello:regenerate_ueber_certs`
 
+STDOUT.puts "Restarting puppet services"
+`service puppet restart`
+`service puppetserver restart`
```

Comment:
@johnpmitsch you might want to use `/sbin/system ...`. I just fixed a bug with katello-backup where the system alias in EL7 doesn't work properly if you run it from a script (i.e. a cron job that runs katello-backup). It's worth considering. https://bugzilla.redhat.com/show_bug.cgi?id=1330757